docs/pages/install/macos.mdx

```diff
@@ -2,36 +2,74 @@
+3. In the list of candidates, copy the link to `pelican_Darwin_x86_64.tar.gz` if you select **AMD64**, or `pelican_Darwin_arm64.tar.gz` if you select **ARM64**.
 
-4. Change the following command with the link to the binary you copied in the previous step and run the command
+4. Change the following command with the link to the binary you copied in the previous step and run the command:
 
     ```bash
     wget <replace-with-the-link-you-copied>
     tar -zxvf pelican_Darwin_arm64.tar.gz // ARM user
     ```
 
-    Note that you need to replace `pelican_Darwin_arm64.tar.gz` with `pelican_Darwin_x86_64.tar.gz` if you are running an Intel machine.
+    Note that you need to replace `pelican_Darwin_arm64.tar.gz` with `pelican_Darwin_x86_64.tar.gz` if you are running an X86 machine.
 
-    Example to install Pelican binary for an Intel machine:
+    Example to install Pelican binary for an ARM machine:
 
     ```bash
-    curl -O https://github.com/PelicanPlatform/pelican/releases/download/v7.5.8/pelican_Darwin_x86_64.tar.gz
-    tar -zxvf pelican_Darwin_x86_64.tar.gz
+    curl -O https://github.com/PelicanPlatform/pelican/releases/download/v7.5.8/pelican_Darwin_arm64.tar.gz
+    tar -zxvf pelican_Darwin_arm64.tar.gz
     ```
 
-5. At this step, you don't actually install Pelican, you just extract the binary. It is recommended that you add Pelican binary to your `PATH` environment variable to allow Pelican to be called from your command line.
+5. The above command extracted the binary from the tar file. You may run the binary in the current folder, but it is recommended that you add Pelican binary to your `PATH` environment variable to allow `pelican` to be called directly from your command line.
 
-## Verify Pelican is installed
+    * Run Pelican binary from the download folder:
 
-Refer to the [previous section](./rhel.mdx#verify-pelican-is-installed) on how to verify that Pelican is successfully installed.
+    ```bash
+    $ cd pelican_Darwin_arm64 # Go to the binary folder
+    $ ./pelican --version # Run Pelican binary
+
+    Version: 7.5.8
+    Build Date: 2024-03-01T18:13:00Z
+    Build Commit: d260a07d3b057d19b7fdd36125f91a8768531258
+    Built By: goreleaser
+    ```
+
+    * Add Pelican binary to your `PATH` for the current terminal
+
+    ```bash
+    $ cd pelican_Darwin_arm64 # Go to the binary folder
+    $ export PATH=$PWD:$PATH # Add current folder to the PATH
+    $ pelican --version # Run Pelican binary
+
+    Version: 7.5.8
+    Build Date: 2024-03-01T18:13:00Z
+    Build Commit: d260a07d3b057d19b7fdd36125f91a8768531258
+    Built By: goreleaser
+    ```
+
+    * Add Pelican binary to your `PATH` permanently
+
+    ```bash
+    $ cd pelican_Darwin_arm64 # Go to the binary folder
+    $ echo 'export PATH=$PWD:$PATH' >> ~/.bashrc # Add the current folder to your .bashrc file
+    $ source ~/.bashrc # Apply the change
```

Comment:
My mac says the default shell is now `zsh`. Are changes to `~/.bashrc` picked up by zsh, or do we need to include an additional set of instructions? (the only difference would be adding it to `~/.zshrc` instead)

Reply:
Thanks for reminding me! I'll do `~/.zshrc` instead. I copied that from Linux but I think macOS uses zsh years ago so I'll default to zsh